### DISCUSSION TITLE
Keyboard improvement

### DISCUSSION THREAD

**Discussion by 4gra** [isontheline/pro.webssh.net#398] [Ideas]
I love the mini keyboard extension, which makes it much easier to use special keys, but I would love if the keyboard could be improved/ customised. For example, I use screen/tmux and it's really useful to be able to bind a key to a control sequence (C-a) to switch screens. Do you have plans to allow keyboard customisation?

I also have a tiny issue with the way the keys are rendered, I find it hard to read the vertical "C t r l" etc and think there might be a better way (for example "C-" or "a-", or the ^ or "option" symbols on an alternate background colour maybe?)

**Comment by isontheline** [comment]
Hello @4gra 👋

Yes there are multiples issues for keyboard enhancements :
* https://github.com/isontheline/pro.webssh.net/issues/215#issuecomment-907865595
* https://github.com/isontheline/pro.webssh.net/issues/350
* https://github.com/isontheline/pro.webssh.net/issues/293

I think it's now time to allow users to implement their own custom keys

**Reply by 4gra** [reply]
Thank you! I didn't think to look at the issues, sorry about that and I'm glad it's on the list.

**Reply by isontheline** [reply]
@4gra No problem, I love to hear that keyboard customization is a wanted feature 🙏

Have a nice day ☀️
